### DISCUSSION TITLE
how to enable git enterprise  licensing  with azure subscription

### DISCUSSION THREAD

**Discussion by MSCOEGithub** [community/community#136843] [Enterprise]
### Select Topic Area

Question

### Body

how to enable git enterprise  licensing  with azure subscription and add users
![image](https://github.com/user-attachments/assets/b35d1adb-4978-4b86-b977-84c3a48b4337)

how to add the identity provider. 
![image](https://github.com/user-attachments/assets/f7c5a787-b9e3-438f-b8a1-6003a5d64b0c)

we are planning to add github enterprise account and add the people as a enterprise users. would like to know identity provider groups linked between teams and enterprise.

**Comment by ferid333** [comment]
**1. Enabling GitHub Enterprise Licensing with Azure Subscription:**

To use GitHub Enterprise with your Azure subscription, follow these steps:

1. **Purchase GitHub Enterprise License**: Make sure you have a GitHub Enterprise license. You can get it from the GitHub website.

2. **Set Up GitHub Enterprise**: Go to your GitHub Enterprise account settings and follow the instructions to set it up. 

3. **Link with Azure**: 
   - Sign in to your Azure portal.
   - Go to "Azure Active Directory" and find "Enterprise applications."
   - Add a new application and search for "GitHub Enterprise."
   - Follow the prompts to link GitHub Enterprise with Azure.

**2. Adding Users:**

To add users to your GitHub Enterprise account:

1. **Log in to GitHub Enterprise**: Go to your GitHub Enterprise instance and sign in.

2. **Go to User Management**: Navigate to the section where you can manage users.

3. **Add Users**: Enter the details of the users you want to add. You can usually add users manually or invite them via email.

**3. Adding an Identity Provider:**

To add an identity provider:

1. **Log in to GitHub Enterprise**.

2. **Go to Settings**: Find the settings for authentication or security.

3. **Add an Identity Provider**: Look for an option to add or configure an identity provider. This might be under a section like "Authentication" or "Single Sign-On."

4. **Follow the Instructions**: You’ll need to enter details about your identity provider (like Azure AD) to link it with GitHub Enterprise.

**4. Linking Identity Provider Groups:**

To link identity provider groups to your GitHub Enterprise:

1. **In Azure Active Directory**: Create and manage groups that you want to sync with GitHub.

2. **In GitHub Enterprise**: Navigate to the section where you manage teams and permissions.

3. **Map Groups**: Link your Azure AD groups to GitHub teams or enterprise groups. This will allow users in these groups to have the appropriate permissions in GitHub.

**Comment by MSCOEGithub** [comment]
As per below point for adding users do we need purchase license for each user also?  and what will be the cost for each user license

2. Adding Users:
